qemu/tests/larger_buffer_with_none_cache_mode.py

```diff
@@ -46,6 +46,9 @@ def run(test, params, env):
     )
     with open(strace_output_file) as fd:
         for line in fd.readlines():
+            # Skip if none of the strace events are in the line
+            if not any(event in line for event in strace_events):
+                continue
             if int(line.split()[-1]) == 2097152:
```

Comment:
Hi @hellohellenmao 
I come up with a simple way based on the related error info:
```suggestion
            if line.split()[-1].isnumeric() and int(line.split()[-1]) == 2097152:
```
However, your solution is okay for me, depends on you.
Thanks.

Reply:
Updated, thanks